Tasks/Grunt/task.json

```diff
@@ -68,6 +78,52 @@
             "required": true,
             "helpMarkDown": "grunt-cli to run when agent can't find global installed grunt-cli  Defaults to the grunt-cli under node_modules folder of cwd.",
             "groupName": "advanced"
+        },
+        {
+            "name": "publishJUnitResults",
+            "type": "boolean",
+            "label": "Publish to TFS/Team Services",
+            "defaultValue": "true",
+            "groupName": "junitTestResults",
```

Comment:
I did that initially but Atin told me to make these names specific to the tool because we were using one tool only.